### DISCUSSION TITLE
Storybook `--docs` mode

### DISCUSSION THREAD

**Discussion by gfellerph** [swisspost/design-system#1744] [Polls]
Storybook v7 has a docs mode: https://storybook.js.org/docs/7.1/react/writing-docs/build-documentation
![image](https://github.com/swisspost/design-system/assets/1659006/643bf57a-98d7-4ffa-8e1c-714ddf5f452a)

In this mode, all story playgrounds are hidden and only `.mdx` docs are visible in the navigation. This enables a very clean look for the documentation because there is only one navigation entry per component. The stories would still be written the same way, but won't show up in the navigation.

On the other hand, addon functionality that's only displayed in the playground will be disabled or hidden. Examples are the [storybook designs addon](https://storybook.js.org/addons/@storybook/addon-designs/) or the [storybook accessibility addon](https://storybook.js.org/addons/@storybook/addon-a11y). While there might be other ways of adding this functionality to the docs, it most likely means more complexity.

In stories mode, the docs are displayed as well and are the default landing page when clicking a component top-level entry in the navigation.

What are your thoughts, do you prefer functionality or simplicity?

**Comment by oliverschuerch** [comment]
We would be able to add Figma Designs to the docs pages only:

```ts
// component.docs.mdx
import { Figma } from '@storybook/addon-designs/blocks';

<Figma url="{url-to-your-figma-file}"/>
```

![image](https://github.com/swisspost/design-system/assets/9716662/1a1cc581-a16e-492c-9cf8-72d0cd9b37e5)

**Comment by gfellerph** [comment]
As blocks are available for key addons, the team decided to go forward with the docs mode.
